src/nn/graph_builder.ts

```diff
@@ -317,8 +317,26 @@ export class MLGraphBuilder {
   /**
    * [spec](https://webmachinelearning.github.io/webnn/#dom-mlgraphbuilder-build)
    */
-  build(outputs: MLNamedOperands): MLGraph {
-    return MLGraph.buildAndCompile(outputs);
+  async build(outputs: MLNamedOperands): Promise<MLGraph> {
+    return new Promise((resolve, reject) => {
+      let graph;
+      try {
+        graph = MLGraph.buildAndCompile(outputs);
```

Comment:
`buildAndCompile` is async function, missing `await` keyword.

Reply:
Fixed, please take another look at new commit, thanks.